genvm/rewards.go

```diff
@@ -11,15 +11,17 @@ import (
 
 // Config defines the configuration options for Spacemesh rewards.
 type Config struct {
-	GasLimit   uint64
-	BaseReward uint64 `mapstructure:"base-reward"`
+	GasLimit          uint64
```

Comment:
didn't notice this before, should GasLimit also have a json tag?

Reply:
i don't know. the reason why i didn't add a tag for GasLimit is because mempool and vm should use same parameter